File: lua/refactoring/tests/refactor/106/csharp/extract_block/extract_block.expected.cs
namespace Test
{
    class Extract
    {
        public static void foo_bar(INSERT_PARAM_TYPE a)
        {
            int test = 1;
            int test_other = 11;
            for (int x = 0; x < test_other + test; x++) {
                Console.WriteLine(x + " " + a);
            }
        }

        public static void simpleFunction(int a)
        {
            foo_bar(a);
        }
    }
}

File: lua/refactoring/tests/refactor/106/java/extract_block/extract_block.expected.java
class Extract {

public static void foo_bar(int a) {
    int test = 1, test_other = 11;
    for (int x = 0; x < test_other + test; x++) {
      System.out.println(x + " " + a);
    }
}


  public static void simpleFunction(int a) {
    foo_bar(a);
  }
}
